keps/sig-windows/2802-identify-windows-pods-apiserver-admission/README.md

```diff
@@ -0,0 +1,393 @@
+It shouldn't impact already running workloads. This is an opt-in feature since users need to explicitly set the OS parameter in the Pod spec i.e .spec.os field. 
+If the feature is disabled the field is preserved if it was already set in the presisted pod object, otherwise it is silently dropped.
+
+
+###### What specific metrics should inform a rollback?
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+Not yet tested.
+
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+None. This feature will be additive
+
+
+### Monitoring Requirements
+
+
+###### How can an operator determine if the feature is in use by workloads?
+
+By looking at the pod's spec. Some fields like Security Constraint Contexts may not
+be applied to the pod spec if the proper OS has been set.
+
+###### How can someone using this feature know that it is working for their instance?
+
+Windows Pods and Linux Pods with proper OS field set in the pod spec would cause pods
+to run properly on desired OS
+
+
+- [x] Events
+  - Event Reason: Kubelet will send pod admitted/denied event.
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+All the pods which have OS field set in the pod spec should have OS specific constraints/defaults applied.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [x] Other (treat as last resort)
+  - Details: Linux Specific SCCs not available on Windows pod specs and vice-versa
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+
+### Dependencies
+
+
+###### Does this feature depend on any specific services running in the cluster?
+None
+
+
+### Scalability
+
+
+
+###### Will enabling / using this feature result in any new API calls?
+No
+
+###### Will enabling / using this feature result in introducing new API types?
+Yes
+
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+No
+
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+It increases the size of Pod object since a new string field is added.
+
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+No
+
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+No
+
+### Troubleshooting
+
+
+###### How does this feature react if the API server and/or etcd is unavailable?
```

Comment:
i guess we need to fill these out?...

Reply:
> i guess we need to fill these next several q's out?...

he deleted the template comments, but technically only for GA I think.

Reply:
That's correct. We can fill these questions when we GA this feature.